runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/graph/LengthPrefixUnknownCodersTest.java

```diff
@@ -143,5 +143,23 @@ public void test() throws IOException {
     assertEquals(expected,
         CoderTranslation.fromProto(updatedCoderProto.getCoder(),
             RehydratedComponents.forComponents(updatedCoderProto.getComponents())));
+
+    // Test that the application of #forCoder is idempotent
+    String updatedCoderId =
```

Comment:
In my previous comment, I was trying to say not to modify the contents of test(), and to just add the additional coder conversions to data(). Note that they already contain some idempotency checks.

Reply:
In fact, I'm pretty ok with the latter three tests demonstrating that this is an idempotent operation, if you're happy.

Reply:
I'm also happy. Will close PR.